.github/workflows/main.yml

```diff
@@ -131,6 +118,65 @@ jobs:
           fi
 
 
+  test_modules:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ 3.6, 3.7 ]
+        torch-version: [ 1.5.0, 1.6.0 ]
+        tensorflow-version: [ 1.15.0 ]
```

Comment:
Are the modules here also tested in the main test? If so, there is no need to have all the python, pytorch versions here.